config/config/wc-config.yaml

```diff
@@ -54,6 +54,7 @@ user:
     tolerations: []
     affinity: {}
     topologySpreadConstraints: []
+    # image: quay.io/prometheus/alertmanager:vX.Y.Z
```

Comment:
As user-alertmanager is not part of upstream I think you should copy and paste whats here into its [values.yaml](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile.d/charts/user-alertmanager/values.yaml) file.
Just so it exists there aswell.

Reply:
Like 1e44ceff9206996fd4cc0829cf3481482fd3f102 which is what the template should produce, or something commented out?

Reply:
Commented out, as it will otherwise take the value from there if the value is not set in wc-config.yaml.

Reply:
Done